src/cloud/project/new-relic.md

```diff
@@ -268,55 +274,77 @@ Prerequisites
 -  Verify that your Cloud environment is [connected to New Relic](#connect-to-new-relic)
 
 {:.procedure}
-To review Adobe-generated alert policies:
+To review the Magento managed alert policies:
+
+1. Use your New Relic credentials to [log in to your New Relic account][New Relic login].
+
+1. Open your application.
 
-1. Use your Project Owner New Relic credentials to [log in to your New Relic account][New Relic login].
+   -  In the top navigation menu, click **APM** to open the application list.
 
-1. From the navigation menu, select  **Alerts**  > **Alert Policies**.
+   -  Click the application name to open the analytics dashboard.
 
-1. In the Search Policies field, search for _Adobe Generated Policy_.
+1. In the top navigation menu, click **Alerts & AI** to open the _Applied Intelligence_ page.
 
-1. You should see a set of policies like the ones below:
+1. In the Search Policies field, search for _Managed alerts_.
 
-   ![Generated alert policies]({{ site.baseurl }}/common/images/cloud/cloud-newrelic-alert-policies.png){:width="650px"}
+1. You should see the _Managed Alerts for Magento Commerce_ policy.
+
+   ![Generated alert policies]({{ site.baseurl }}/common/images/cloud/cloud-new-relic-magento-managed-alerts.png){:width="650px"}
 
    {:.bs-callout-info}
-   If you do not see these alert policies, submit a Magento support ticket. Include your project ID in the ticket. These policies are available on Pro and Starter Production environments only.
+   If you do not see this alert policy, submit a Magento support ticket. Include your project ID in the ticket. The policy is available on Pro and Starter Production environments only.
+
+1. Click **Managed Alert policies for Magento Commerce** to review the alert conditions defined in the policy.
```

Comment:
If it is the  "Managed Alerts policy" , then why is it the "Managed Alert policies" in the ui? I'm confused 😖 
```suggestion
1. Click **Managed Alerts for Magento Commerce** to review the alert conditions defined in the policy.
```